lib/wallet/integration/scenarios/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -399,27 +398,6 @@ spec = describe "SHELLEY_STAKE_POOLS" $ do
                             (`shouldBe` walletRewards)
                         ]
 
-            -- Listing stake keys shows
-            request @(ApiStakeKeys n) ctx (Link.listStakeKeys src) Default Empty
-                >>= flip
-                    verify
-                    [ expectField (#_foreign) (`shouldBe` [])
-                    , expectField
-                        (#_ours)
-                        ( \case
-                            [acc] -> do
-                                (acc ^. #_stake) .> ApiAmount 0
-                                acc
-                                    ^. (#_delegation . #active . #status)
-                                        `shouldBe` Delegating
-                                acc
-                                    ^. (#_delegation . #active . #target)
-                                        `shouldBe` (Just (ApiT pool))
-                            _ -> expectationFailure "wrong number of accounts in \"ours\""
-                        )
-                    , expectField (#_none . #_stake) (.> ApiAmount 0)
```

Comment:
Failed here, IIRC. Actually, I'm not sure why this wallet would have any funds not associated with a stake key... But 
`listStakeKeys` should arguably be removed, and doesn't need to have an expectation here

Reply:
Reverted — related to another change now not apart of this PR.